charts/nginx-gateway-fabric/templates/gatewayclass.yaml

```diff
@@ -4,6 +4,12 @@ metadata:
   name: {{ .Values.nginxGateway.gatewayClassName }}
   labels:
   {{- include "nginx-gateway.labels" . | nindent 4 }}
+  {{- if .Values.nginxGateway.gatewayClassAnnotations }}
+  annotations:
+    {{- range $key, $value := .Values.nginxGateway.gatewayClassAnnotations }}
+    {{ printf "%s: %s" $key $value }}
+    {{- end }}
```

Comment:
Out of curiosity, what happens if we don't do the range here, and just include the field directly? How does it render?

Reply:
IMHO, it gives more control on format of inserted values. But, I can insert it as is

Reply:
I think I would use `toYaml` to just insert it directly...